### DISCUSSION TITLE
Any plans to start using TypeScript?

### DISCUSSION THREAD

**Discussion by huyz** [Budibase/budibase#2849] [Ideas]
**Describe the feature request**
I'm evaluation different open-source low-code platforms, which give the option of forking and customizing. I'm wondering if there are plans for budibase to start using typescript internally?

**Comment by mjashanks** [comment]
We have started using Typescript server-side, but have no plans to use it on the frontend.

**Comment by mike12345567** [comment]
Hi @huyz - I've converted this into a discussion rather than an issue.

As for Typescript support, we currently partially use it in our server package, found [here](https://github.com/Budibase/budibase/tree/develop/packages/server). We plan to expand our usage of Typescript in the backend, especially for areas that benefit from structural typing but there are no plans for the frontend.
